web-common/src/features/entity-management/resource-selectors.ts

```diff
@@ -6,11 +6,12 @@ import {
 } from "@rilldata/web-common/runtime-client";
 
 export enum ResourceKind {
-  Source = "source",
-  Model = "model",
-  MetricsView = "metricsview",
-  // TODO: do a correct map based on backend code
+  ProjectParser = "rill.runtime.v1.ProjectParser",
+  Source = "rill.runtime.v1.SourceV2",
+  Model = "rill.runtime.v1.ModelV2",
+  MetricsView = "rill.runtime.v1.MetricsViewV2",
 }
```

Comment:
It'd be nice if Orval generated these types. Any chance you know if there's something missing from the backend API definition?

Reply:
The enum sits in go rather than the proto definition. @begelundmuller Any reason to not add it to the proto?